package.json

```diff
@@ -45,9 +45,11 @@
     "@types/tar": "^6.1.12",
     "@types/url-join": "^4.0.3",
     "debug": "^4.3.4",
+    "fetch-blob": "^4.0.0",
+    "formdata-polyfill": "^4.0.10",
```

Comment:
These where included transitively before, but need to be referenced explicitly now.